dd-sdk-android/src/main/kotlin/com/datadog/android/telemetry/internal/TelemetryEventHandler.kt

```diff
@@ -235,13 +234,41 @@ internal class TelemetryEventHandler(
                     batchSize = coreConfig?.batchSize?.windowDurationMs,
                     batchUploadFrequency = coreConfig?.uploadFrequency?.baseStepMs,
                     mobileVitalsUpdatePeriod = rumConfig?.vitalsMonitorUpdateFrequency?.periodInMs,
-                    useTracing = traceFeature != null && GlobalTracer.isRegistered(),
+                    useTracing = traceFeature != null && isGlobalTracerRegistered(),
                     trackNetworkRequests = trackNetworkRequests
                 )
             )
         )
     }
 
+    private fun isGlobalTracerRegistered(): Boolean {
```

Comment:
Do you think this is really needed as long as the `traceFeature!=null` ?

Reply:
Let's keep it as it is for now, because originally both feature check and tracer check were done.